tests/install_test.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package tests
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"os/user"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"code.gitea.io/gitea/tests/internal/utils"
+)
+
+// The HTTP port listened by the Gitea server.
+const ServerHTTPPort = "3001"
+
+const _RetryLimit = 10
+
+func makeSimpleSettings(user, workdir, port string) map[string][]string {
+	return map[string][]string{
+		"db_type":        {"SQLite3"},
```

Comment:
Doesn't this need a build tag to work ?
Any chance to test other databases too ?